### PR TITLE
add alerts to kube-burner dash

### PR DESCRIPTION
**Pull request by mohit-sheth**
### Description
add alerts to kube-burner dash

### CODE REVIEW

**Review by jtaleric — APPROVED** [review]
LGTM!

File: roles/kube-burner/files/metrics-aggregated.yaml [hunk 1]
```diff
@@ -182,3 +182,6 @@ metrics:
   
   - query: go_gc_duration_seconds{job=~"apiserver|api|etcd",quantile="1"}
     metricName: goGCDurationSeconds
+
+  - query: topk(10,sum(ALERTS{severity!="none"}) by (alertname, severity))
```
Comment (rsevilla87) [inline roles/kube-burner/files/metrics-aggregated.yaml:186]: Maybe we can remove sum from the query to have all labels?
Usint the original query:
image

Without sum
image

**Review by rsevilla87 — APPROVED** [review]
LGTM!

**Pull request merged**